.stylelintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "stylelint-config-standard"
+}
```

Comment:
I'd add EOL where missing

Reply:
End-of-file newlines [are enabled](https://github.com/stylelint/stylelint-config-standard/blob/master/index.js#L78).  ... or were you referring to some other missing EOLs?

Reply:
eslint doesn’t lint JSON files. This file lacks a EOL newline